src/sentry/integrations/utils/code_mapping.py

```diff
@@ -267,33 +267,11 @@ def list_file_matches(self, frame_filename: FrameFilename) -> List[Dict[str, str
                         "repo_name": repo_tree.repo.name,
                         "repo_branch": repo_tree.repo.branch,
                         "stacktrace_root": f"{frame_filename.root}/",
-                        "source_path": self._get_code_mapping_source_path(file, frame_filename),
+                        "source_path": _get_code_mapping_source_path(file, frame_filename),
```

Comment:
Moving the function outside of the class makes it easier (or just cleaner) to patch in the tests.